src/collective/documentgenerator/tests/test_pod_template.py

```diff
@@ -17,8 +19,12 @@
 from zope.component import queryAdapter
 from zope.component import queryMultiAdapter
 from zope.interface import Interface
+from zope.interface import Invalid
 
-import unittest
+
+import unittest2 as unittest
```

Comment:
Not needed, this is for Python 2.4 compatibility, use unittest and remove everything about unittest2